src/include/sof/samples/audio/c_models_inc/config.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Mihai Despotovici <mihai.despotovici@nxp.com>
+ */
+
+#ifndef __CONFIG_H__
+#define __CONFIG_H__
+
+
+// RAW AUDIO DATA CONFIGURATION
+#define MS_TO_SAMPLES(SAMPLERATE, TIME) ((SAMPLERATE) * (TIME) / 1000)
+#define SIZE_FROM_STRIDE_SIZE(STRIDE, SIZE, NO_WINDOWS) ((NO_WINDOWS - 1) * STRIDE + SIZE)
+
+#define CONFIG_SAMPLERATE 16000
```

Comment:
These will need to use a prefix in order to namespace the config options. e.g. FEATURE_CONFIG_ITEM

Reply:
Also we sould eventually move these to be _actual_ config options in KConfig.

Reply:
@paulstelian97 we can live with that for now. Imagine trying to configure all of these options from config :)